.github/workflows/update-snapshot.yml

```diff
@@ -240,9 +240,22 @@ jobs:
       - name: Build Test
         working-directory: ./packages/react-composites
         run: rushx build:e2e
-      - name: Update snapshot
-        id: updatesnapshots
-        run: rushx test:e2e:update
+      - name: Update chat snapshot
+        id: update-chat-snapshots
+        run: rushx test:e2e:chat:update
+        working-directory: ./packages/react-composites
+        env:
+          CONNECTION_STRING: ${{ secrets.CONNECTION_STRING }}
+      - name: Update calling snapshot
+        id: update-calling-snapshots
+        run: rushx test:e2e:call:update
+        working-directory: ./packages/react-composites
+        env:
+          CONNECTION_STRING: ${{ secrets.CONNECTION_STRING }}
+      - name: Update meeting snapshot
+        id: update-meeting-snapshots
+        if: ${{ matrix.flavor != 'stable' }}
```

Comment:
Why the special handling? Because there are 0 tests on stable, and there is no (easy) way for jest to accept an empty test suite?

Can you add a comment here that this check should be removed when MeetingComposite goes stable. We're likely to forget to update this...

Reply:
Because when we are doing conditional compiling, we don't do it for *.test.ts, so in the test.ts we need to read environment variables to determine whether it is under stable flavor, we can change something there if we want (but we don't even want to spin up the test server, so I guess skip the test for stable would be the best choice)

Reply:
I think updating the test file is a better choice, if it is possible. Having an ad-hoc condition like this is dangerous because we'll likely forget about it when the feature goes stable (and... never test it on stable)